cassandra/src/main/scala/com/stratio/crossdata/connector/cassandra/statements/CreateTableStatement.scala

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stratio.crossdata.connector.cassandra.statements
+
+import com.datastax.spark.connector.types._
+import com.stratio.crossdata.connector.cassandra.DefaultSource.CassandraDataSourcePrimaryKeyNameProperty
+import org.apache.spark.sql.types.ByteType
+import org.apache.spark.sql.types.IntegerType
+import org.apache.spark.sql.types.StructField
+import org.apache.spark.sql.types.ArrayType
+import org.apache.spark.sql.types.StringType
+import org.apache.spark.sql.types.BinaryType
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.types.ShortType
+import org.apache.spark.sql.types.LongType
+import org.apache.spark.sql.types.{BooleanType => SparkSqlBooleanType}
+import org.apache.spark.sql.types.{DataType => SparkSqlDataType}
+import org.apache.spark.sql.types.{DecimalType => SparkSqlDecimalType}
+import org.apache.spark.sql.types.{DoubleType => SparkSqlDoubleType}
+import org.apache.spark.sql.types.{FloatType => SparkSqlFloatType}
+import org.apache.spark.sql.types.{MapType => SparkSqlMapType}
+import org.apache.spark.sql.types.{TimestampType => SparkSqlTimestampType}
+
+object CreateTableStatement {
+
+  val PRIMARY_SINGLE = 1
+  val PRIMARY_COMPOSED = 2
+  val PRIMARY_AND_CLUSTERING_SPECIFIED = 3
+
+}
+
+case class CreateTableStatement(tableName: String,
+                                schema: StructType,
+                                options: Map[String, String]
+                               ) {
+
+  override def toString(): String = {
+    val cqlCommand = StringBuilder.newBuilder
+    cqlCommand.append("CREATE TABLE ")
+
+    if (ifNotExists) {
+      cqlCommand.append("IF NOT EXISTS ")
+    }
+
+    cqlCommand.append(keyspace).append(".").append(tableName)
```

Comment:
What happens if keyspace doesn't exist? Should there be a previous condition checking if that keyspace exists?